test/utils/evaluate-testcase.js

```diff
@@ -69,6 +69,13 @@
         keys.forEach(function (key) {
             if (o.hasOwnProperty(key)) {
                 result[key] = sortedObject(o[key]);
+
+                // Nullify regex value for ES6 regex flags
+                if (key === 'value' && o.hasOwnProperty('regex')) {
+                    if ('uy'.indexOf(o.regex.flags) >= 0) {
```

Comment:
I'm hesitant to completely ignore the regex object for everything else, since there is a value doing that. I agree on the limited scope of the check, I'll expand it to account for cases whenever `u` or `y` is present.